installinator/src/write.rs

```diff
@@ -334,7 +351,8 @@ impl<'a> ArtifactWriter<'a> {
                                     *progress =
                                         DriveWriteProgress::ControlPlaneFailed;
                                 }
-                                WriteComponent::Unknown => {
+                                WriteComponent::Unknown
+                                | WriteComponent::MeasurementCorpus => {
```

Comment:
This doesn't seem like it should be `unreachable!()` - we could fail to write the measurement corpus, right?